src/commands/utility/tinyurl.js

```diff
@@ -0,0 +1,25 @@
+const { Command } = require('../../')
+const fetch = require('node-fetch')
+
+module.exports = class Tinyurl extends Command {
+  constructor (client) {
+    super(client, {
+      name: 'tinyurl',
+      aliases: ['short', 'shorturl'],
+      category: 'utility',
+      parameters: [{
+        type: 'string',
+        full: true,
+        missingError: 'commands:tinyurl.missingSentence'
+      }]
+    })
+  }
+
+  async run ({ t, author, channel }, text) {
+   channel.startTyping()
+   fetch(`http://tinyurl.com/api-create.php?url=${encodeURIComponent(text)}`).then(shorturl => {
```

Comment:
use await and handle errors like this:
```
try {
  const shortUrl = await fetch(`http://tinyurl.com/api-create.php?url=${encodeURIComponent(text)}`)
  channel.send(shortUrl)
} catch (e) {
  throw new CommandError(t('commands:tinyurl.shortingError'))
}
```
you will need of course to import CommandError and create that string.
Also would be nice if you use SwitchEmbed too